editor/create_dialog.cpp

```diff
@@ -554,7 +555,17 @@ void CreateDialog::_item_selected() {
 		return;
 	}
 
-	help_bit->set_text(EditorHelp::get_doc_data()->class_list[name].brief_description);
+	if (EditorHelp::get_doc_data()->class_list.has(name) && !EditorHelp::get_doc_data()->class_list[name].brief_description.empty()) {
+		help_bit->set_text(EditorHelp::get_doc_data()->class_list[name].brief_description);
+		// Display both class name and description, since the help bit may be displayed
+		// far away from the location (especially if the dialog was resized to be taller).
+		help_bit->set_text(vformat("[b]%s[/b]: %s", name, EditorHelp::get_doc_data()->class_list[name].brief_description.strip_edges()));
```

Comment:
Unlike `master`, I had to use `strip_edges()` here to prevent stay line breaks from being included.